src/devices_models/device_constructors/storage_constructor.jl

```diff
@@ -353,6 +354,7 @@ function construct_device!(
     add_variables!(optimization_container, ActivePowerInVariable, devices)
     add_variables!(optimization_container, ActivePowerOutVariable, devices)
     add_variables!(optimization_container, EnergyVariable, devices)
+    add_variables!(optimization_container, ReserveVariable, devices)
```

Comment:
shouldn't this formulation have an energy target constraint?

Reply:
I'll be adding that as per our discussions